### PR TITLE
Ensure some files are not exported

### PR DESCRIPTION
**Pull request by lucasmichot**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
We should add the tests in here too.

**Comment by lucasmichot** [comment]
@vlucas can I add some more files ?

**Comment by vlucas** [comment]
Sure - just push to this branch agin

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+tests/ export-ignore
```
Comment (GrahamCampbell) [inline .gitattributes]: shouldn't be a slash?

**Pull request merged**